src/server/game/Entities/Player/Player.cpp

```diff
@@ -1417,6 +1417,9 @@ bool Player::TeleportTo(uint32 mapid, float x, float y, float z, float orientati
         if (!(options & TELE_TO_NOT_LEAVE_COMBAT))
             CombatStop();
 
+        //remove auras before teleporting...
+        RemoveAurasWithInterruptFlags(AURA_INTERRUPT_FLAG_TELEPORTED | AURA_INTERRUPT_FLAG_MOVE | AURA_INTERRUPT_FLAG_TURNING);
```

Comment:
I don't thinks it's a good change. It will remove auras with these flags on teleport on the same map, e.g. blink or shadowstep.

Reply:
Then I could make it removed only  if teleport to more than a specified distance (like 100) to avoid it being removed from spells such as blink ?

Reply:
More more:
I Will look forward in dbcs to find out if such spells have this aura interrupt flag, which seems at my sense illogic

Reply:
@UltraNix I moved it to the clause where it will only be removed if teleports on another map OR teleport at least 100 yards away.
I overall deleted the other flags I should not remove in this case.

Sorry for my confusion about blink and shadowstep, if you read my previous messages before I deleted them. I did not understand and thought you were saying this edit was gonna remove blink aura or shadowstep aura, I was really confused about it (x
I see better now, you were talking about blink or shadowstep were gonna remove auras they probably shouldn't. So it is avoid by the 100yards condition then (neither blink or shadowstep can send the player so far away).